src/test/java/de/tum/cit/aet/artemis/exam/StudentExamIntegrationTest.java

```diff
@@ -2236,13 +2240,16 @@ void testGradedFinalExamSummaryWithPlagiarismAndNotParticipatedBonusExamAsStuden
     @Test
     @WithMockUser(username = TEST_PREFIX + "instructor1", roles = "INSTRUCTOR")
     void testDeleteExamWithStudentExamsAfterConductionAndEvaluation() throws Exception {
+        final var baseTime = ZonedDateTime.now();  // use a base time rather than repeated 'ZonedDateTime.now()' calls
+        // to prevent potential time-related flakiness
```

Comment:
I would write it in one line, no super strong opinion
```suggestion
        final var baseTime = ZonedDateTime.now();  // use a base time rather than repeated 'ZonedDateTime.now()' to prevent potential time-related flakinesscalls
```